Makefile

```diff
@@ -79,12 +79,21 @@ endif
 
 .PHONY: __bake
 __bake: upload-code upload-data upload-notebooks
-	echo "#!/usr/bin/env bash" > jupyter.sh
-	echo "jupyter notebook --no-browser --ip=0.0.0.0 --allow-root \
-	    --NotebookApp.token= --notebook-dir=/project/notebooks" >> jupyter.sh
-	$(NEURO) cp jupyter.sh $(PROJECT_PATH_STORAGE)/jupyter.sh
+	echo "#!/usr/bin/env bash" > /tmp/jupyter.sh
+	echo "jupyter notebook \
+            --no-browser \
+            --ip=0.0.0.0 \
+            --allow-root \
+            --NotebookApp.token= \
+            --NotebookApp.default_url=/notebooks/project/notebooks/demo.ipynb \
```

Comment:
Still, shouldn't this URL be `/project/notebooks/demo.ipynb`?

Reply:
nope :)

Reply:
jupyter's URLs for notebooks must start with `/notebooks/` and then path to the notebook. It works :)